frontend/src/views/producer/sign-in/SignInForm.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import Form, { InputField } from '../../../components/form';
+import Button from '../../../components/button';
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+  align-items: center;
+  width: 25rem;
+`;
+const SignInButton = styled(Button)<any>`
+  &&& {
+    margin: 2rem 0;
+    align-self: flex-end;
+    background-color: ${props => props.theme.palette.lightGreen};
+    color: white;
+  }
+`;
+
+export default class SignInForm extends React.Component {
+  public render() {
+    return (
+      <Wrapper>
+        <Form>
+          <InputField label="Nimi" />
+          <InputField label="Password" type="password" />
+        </Form>
+        <SignInButton>Kirjaudu</SignInButton>
```

Comment:
You're right. Makes more sense that way. I will incorporate this change in my next PR because everything is in a state of WIP rn, and merging this with my current WIP will cost more time